.gitlab-ci.yml

```diff
@@ -112,19 +118,66 @@ publish-snap:
   allow_failure:                   true
   <<:                              *collect_artifacts
 
-publish-awss3:
-  stage:                           publish
+publish:onnet:update:
+  stage:                           publish-onchain
   only:                            *releaseable_branches
-  cache: {}
+  cache:                           {}
   dependencies:
     - build-linux
     - build-darwin
     - build-windows
+    - publish:awss3:release
```

Comment:
can we use dashs rather than colons?

Reply:
Wanted to move to colons in the final CI version, what's wrong with them? I really do not care, just thought it's nicer.

Reply:
@TriplEight colons don't work for Windows jobs. So either we have some mix, or it's dashes-only.

Reply:
AFAIR it's related to Windows FS limitations: gitlab-runner wants to create a file with the job name somewhere, and if it fails — the whole job fails.

Reply:
oh, didn't know, will definitely fix later.